app/views/producer_mailer/order_cycle_report.text.haml

```diff
@@ -0,0 +1,28 @@
+Dear #{@producer.name},
+\
+We now have all the consumer orders for the food drop on #{@distribution_date}.
+Please deliver to #{@coordinator.address.address1}, #{@coordinator.address.city}, #{@coordinator.address.zipcode} during the regular delivery time. If this is not convenient then please call #{@coordinator.phone}.
+
+Note: If you have to arrange a different delivery day and time, it is requested that you do not come on site during drop off/pick up times.
+
+\
+Orders summary
+================
+\
+Here is a summary of the orders for your products:
+\
+- @line_items.each_pair do |variant, data|
+  #{variant.sku} #{raw(variant.product.supplier.name)} #{raw(variant.product.name)} #{raw(variant.options_text)} (QTY: #{data[:quantity]}) @ #{data[:line_item].single_money} = #{variant.display_amount}
```

Comment:
I haven't tested this myself, but I'm not sure about the money calculations here. In particular, shouldn't `variant.display_amount` be `data[:line_item].display_amount`? This would be a good one to write a spec for to check if it's doing the right thing.